VAMobile/src/screens/HealthScreen/Pharmacy/RefillScreens/RefillRequestSummary.tsx

```diff
@@ -160,7 +160,15 @@ function RefillRequestSummary({ navigation, route }: RefillRequestSummaryProps)
               {rxNumber}
             </TextView>
           </Box>
-          <VAIcon {...vaIconProps} />
+          <Box
+            justifyContent={'center'}
+            alignItems={'center'}
+            backgroundColor={request.submitted ? 'completedPhase' : 'buttonDestructiveActive'}
+            borderRadius={indicatorDiameter > 24 ? 24 : indicatorDiameter}
```

Comment:
Optional, could use a const since this is repeated 3x

Reply:
I think we can leave this as is.